configure_pretraining.py

```diff
@@ -115,11 +115,14 @@ def __init__(self, model_name, data_dir, **kwargs):
     # paper for the full hyperparameters
     # else:
     #   self.max_seq_length = 512
-    #   self.learning_rate = 2e-5
+    #   self.learning_rate = 2e-4
     #   if self.model_size == "base":
+    #     self.embedding_size = 768
```

Comment:
this isn't strictly needed because the embedding size is set to the hidden size when self.embedding_size=None, but perhaps it's better to explicitly set it anyway